scripts/push_to_gh_pages.sh

```diff
@@ -1 +1,5 @@
+#!/bin/bash
+PREVIOUS_CURRENT_DIR=$(pwd)
+cd ..
 git checkout gh-pages; git merge master; git push; git checkout master
```

Comment:
GitHub can host some web pages, if you simply push them in a gh-pages branch.
I do that on each major release. The result is http://kiwix.github.io/kiwix-html5
It is mainly for testing purpose